.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)